lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -124,19 +144,18 @@ window.runLighthouseForConnection = function(connection, url, options, requested
   lighthouseIsRunning = true;
   updateBadgeUI(url);
 
-  // Run Lighthouse.
-  return Runner.run(connection, runOptions)
-    .then(result => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      filterOutArtifacts(result);
-      return result;
-    })
-    .catch(err => {
-      lighthouseIsRunning = false;
-      updateBadgeUI();
-      throw err;
-    });
+  return Runner.run(connection, runOptions) // Run Lighthouse.
```

Comment:
It's just a proper 4-space indent. https://github.com/GoogleChrome/lighthouse/pull/1492/files?w=1